src/Installers/Windows/AspNetCoreModule-Setup/ANCMV2/aspnetcoremodulev2.wxs

```diff
@@ -62,6 +62,7 @@
 
         <Property Id="ARPCONTACT">Microsoft Corporation</Property>
         <Property Id="ARPPRODUCTICON" Value="AspNetCoreModule.ico" />
+        <Property Id="DISABLESHAREDCONFIGCHECK"  Value="0"/>
```

Comment:
Should we invert the value and rename the property to `SHAREDCONFIGCHECK` ?

Reply:
Default should be check

Reply:
It doesn't make sense to do "OPT_SHARED_CONFIG_CHECK" and default the value to 1 because Opt implies you are opting into the behavior rather than opting out. Going to change it to OPT_NO_SHARED_CONFIG_CHECK for now.